docs/en/week06/06-3.md

```diff
@@ -0,0 +1,203 @@
+
+If you do a PCA over the latent space, you will have the words grouped by semantics like shown in this graph.
+![](https://i.imgur.com/CYytabM.jpg)
+If we zoom in, we will see that the in the same location there are all the months, like January and November. 
+![](https://i.imgur.com/B6jRSib.png)
+If you focus on a different region, you get words like "a few days ago " the next few months etc.
+![](https://i.imgur.com/LVqS9xi.png)
+From these examples, we see that different location will have some specific common meaning.
+
+This graph below showcases how how by training this kind of network will pick up on some semantics features. For exmaple in this case you can see there is a vector connecting man to woman and another between king and queen, which means woman minus man is going to be equal to queen minus king. You will get the same distance in this embeddings space applied to cases like male-female. Another example will be walking to walked and swimming to swam. You can always apply this kind of specific linear transofmation going from one word to another or from country to capital.
+![](https://i.imgur.com/ZmTZETy.png)
+
+The last case is sequence to sequence. In this network, as you start feeding inside input the network starts getting output. An example of this type of architecture is T9, if you remeber using Nokia phone, you will get text suggestions as you are typing things through. Another example is speech to captions. One of the cool example is this RNN-writer. When you start typing "the rings of saturn glittered while", it suggests the following "two men looked at each other". This network was trained on some sci-fi novels so that you can just type somthing and let it make suggestions to help you write a book. One more example is shown below. You input the top prompt and then this network will try to complete the rest.
+![](https://i.imgur.com/OinKjjV.jpg)
+
+
+## Back Propagation through time (14:00 - 23:50) 
+#### 1. Model architecture
+
+In order to perform training on RNN, backpropagation through time (BPTT) is supposed to be used. The model architechture of RNN is given in figure below. The left design uses loop representation while the right figure unfolds the loop into a row over time. 
+
+![](https://i.imgur.com/BYmRGQv.jpg)
+Hidden representations are stated as
+$$\begin{aligned}
```

Comment:
This does not render because there is no empty line before the math.
Check https://atcold.github.io/pytorch-Deep-Learning/en/about/